From 37de3af53b97bebb4b6ef8c416ffb508fed94a6e Mon Sep 17 00:00:00 2001
From: Chris Mayo <aklhfex@gmail.com>
Date: Tue, 7 Jun 2016 20:01:55 +0100
Subject: [PATCH 2/7] fix desktop Categories entry

Single entry prefixing non-standard categories

(cherry picked from commit a64635049f62234e03344da5be6f8d3bf09e2072)
---
 fritzing.desktop | 3 +--
 1 file changed, 1 insertion(+), 2 deletions(-)

diff --git a/fritzing.desktop b/fritzing.desktop
index 1791c92..5b14537 100644
--- a/fritzing.desktop
+++ b/fritzing.desktop
@@ -7,8 +7,7 @@ Exec=Fritzing
 Icon=fritzing
 Terminal=false
 Type=Application
-Categories=Development;IDE;Electronics;EDA;
+Categories=Development;IDE;Electronics;X-EDA;X-PCB;
 X-SuSE-translate=false
 StartupNotify=true
-Categories=PCB;
 MimeType=application/x-fritzing-fz;application/x-fritzing-fzz;application/x-fritzing-fzp;application/x-fritzing-fzpz;application/x-fritzing-fzb;application/x-fritzing-fzbz;application/x-fritzing-fzm;
-- 
2.8.3

